src/screens/VisualTests/VisualTests.tsx

```diff
@@ -196,12 +201,12 @@ export const VisualTests = ({
   const isRunningBuildStarting =
     runningBuild && !["success", "error"].includes(runningBuild.currentStep);
 
-  return !storyBuild || error ? (
+  return !storyBuild || storyDataIsStale || error ? (
     <NoBuild
       {...{
         error,
-        hasData: !!data,
-        hasStoryBuild: !!storyBuild,
+        hasData: !!data && !storyDataIsStale,
+        hasStoryBuild: !storyBuild,
```

Comment:
```suggestion
        hasStoryBuild: !!storyBuild,
```
Change needed to fix the no build state and stories